src/azure-cli/azure/cli/command_modules/apim/tests/latest/test_apim_scenario.py

```diff
@@ -366,7 +366,65 @@ def test_apim_core_service(self, resource_group, resource_group_location, storag
         self.assertEqual(api_count, 1)
 
         count = len(self.cmd('apim list').get_output_in_json())
-
+        pythonfile = 'gql_schema.gql'
```

Comment:
If all the features of graphql are supported by consumption sku, you can create a new test method that activate a consumption sku and do all the graphql related test over there. It may help you with your future graphql related CLI work since it will take less time to run the test, but it all depends on your preference, you can do whatever you found is more comfortable with

Reply:
I also think is better to create a new test method.
Put all test cases in one test method will make CI getting slower.